.github/workflows/FrontEnd-Lint-Test.yml

```diff
@@ -5,7 +5,7 @@ on: [pull_request]
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why doesn't ubuntu-latest work here anymore?

Reply:
ubuntu-latest has been updated to 22.04, since this update has happened Chromeheadless fails to start causing the build to fail in github actions.